Projects/19-Captsone_01/ispy1/inferential_statistics.py

```diff
@@ -0,0 +1,132 @@
+import pandas as _pd
+import numpy as np
+from scipy.stats import chi2_contingency as _chi2
+from scipy.stats import fisher_exact
+import matplotlib as plt
+import seaborn as sns
+from scipy import stats
+from scipy.stats import fisher_exact
+from sklearn import preprocessing
+from statsmodels.formula.api import ols
+import statsmodels.api as sm
+import matplotlib.pyplot as plt
+
+# CATEGORICAL PREDICTORS and CATEGORICAL OUTCOMES
+def contingency_table(predictor, outcome, dataframe):
+    '''
```

Comment:
I highly recommend using the Sphinx docstring format for future projects:
https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format

Don't worry about it for this project.